src/holochain.rs

```diff
@@ -43,10 +43,6 @@ pub trait AdminCall: std::fmt::Debug + Send + Sync {
         allowed_origins: AllowedOrigins,
         installed_app_id: Option<String>,
     ) -> BoxFuture<'static, HcHttpGatewayResult<u16>>;
-
-    /// Set the admin websocket connection to use.
-    #[cfg(feature = "test-utils")]
```

Comment:
No longer required, the admin socket reconnects and we don't need to simulate it any more